chain/actors/actor_cids.go

```diff
@@ -0,0 +1,326 @@
+
+		case Version4:
+			return builtin4.MultisigActorCodeID, true
+
+		case Version5:
+			return builtin5.MultisigActorCodeID, true
+
+		case Version6:
+			return builtin6.MultisigActorCodeID, true
+
+		case Version7:
+			return builtin7.MultisigActorCodeID, true
+		}
+
+	case PaychKey:
+		switch av {
+
+		case Version0:
+			return builtin0.PaymentChannelActorCodeID, true
+
+		case Version2:
+			return builtin2.PaymentChannelActorCodeID, true
+
+		case Version3:
+			return builtin3.PaymentChannelActorCodeID, true
+
+		case Version4:
+			return builtin4.PaymentChannelActorCodeID, true
+
+		case Version5:
+			return builtin5.PaymentChannelActorCodeID, true
+
+		case Version6:
+			return builtin6.PaymentChannelActorCodeID, true
+
+		case Version7:
+			return builtin7.PaymentChannelActorCodeID, true
+		}
+
+	case PowerKey:
+		switch av {
+
+		case Version0:
+			return builtin0.StoragePowerActorCodeID, true
+
+		case Version2:
+			return builtin2.StoragePowerActorCodeID, true
+
+		case Version3:
+			return builtin3.StoragePowerActorCodeID, true
+
+		case Version4:
+			return builtin4.StoragePowerActorCodeID, true
+
+		case Version5:
+			return builtin5.StoragePowerActorCodeID, true
+
+		case Version6:
+			return builtin6.StoragePowerActorCodeID, true
+
+		case Version7:
+			return builtin7.StoragePowerActorCodeID, true
+		}
+
+	case RewardKey:
+		switch av {
+
+		case Version0:
+			return builtin0.RewardActorCodeID, true
+
+		case Version2:
+			return builtin2.RewardActorCodeID, true
+
+		case Version3:
+			return builtin3.RewardActorCodeID, true
+
+		case Version4:
+			return builtin4.RewardActorCodeID, true
+
+		case Version5:
+			return builtin5.RewardActorCodeID, true
+
+		case Version6:
+			return builtin6.RewardActorCodeID, true
+
+		case Version7:
+			return builtin7.RewardActorCodeID, true
+		}
+
+	case SystemKey:
+		switch av {
+
+		case Version0:
+			return builtin0.SystemActorCodeID, true
+
+		case Version2:
+			return builtin2.SystemActorCodeID, true
+
+		case Version3:
+			return builtin3.SystemActorCodeID, true
+
+		case Version4:
+			return builtin4.SystemActorCodeID, true
+
+		case Version5:
+			return builtin5.SystemActorCodeID, true
+
+		case Version6:
+			return builtin6.SystemActorCodeID, true
+
+		case Version7:
+			return builtin7.SystemActorCodeID, true
+		}
+
+	case VerifregKey:
+		switch av {
+
+		case Version0:
+			return builtin0.VerifiedRegistryActorCodeID, true
+
+		case Version2:
+			return builtin2.VerifiedRegistryActorCodeID, true
+
+		case Version3:
+			return builtin3.VerifiedRegistryActorCodeID, true
+
+		case Version4:
+			return builtin4.VerifiedRegistryActorCodeID, true
+
+		case Version5:
+			return builtin5.VerifiedRegistryActorCodeID, true
+
+		case Version6:
+			return builtin6.VerifiedRegistryActorCodeID, true
+
+		case Version7:
+			return builtin7.VerifiedRegistryActorCodeID, true
+		}
+	}
+
+	return cid.Undef, false
+}
+
+// GetActorCodeIDs looks up a builtin actor's code CID by actor version.
```

Comment:
Comment seems misleading -- it doesn't lookup a specific actor's code CID, but instead returns all builtin actor code CIDs for the given AV?